ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/PTP_TESTS.yml

```diff
@@ -86,6 +86,20 @@ servers:
           enabled: true
           endpoint_role: follower
 
+  - name: Endpoint-with-mpass
+    adapters:
+      - endpoint_ports: [ PCI1, PCI2 ]
+        switch_ports: [ Port-Channel1, Port-Channel2 ]
```

Comment:
This is not how you should configure it. These should be the ethernet intefaces.

Reply:
Changed it to Ethernet.